test/CMakeLists.txt

```diff
@@ -312,19 +317,25 @@ set (destination "${CMAKE_BINARY_DIR}/test/data/tile_src")
 
 ## Test run targets
 foreach(test ${tests} ${cost_tests} ${tyr_tests})
-  add_custom_command(OUTPUT ${test}.log
-    COMMAND
-      LOCPATH=${VALHALLA_SOURCE_DIR}/locales
-      /bin/bash -c "${CMAKE_CURRENT_BINARY_DIR}/${test} >& ${CMAKE_CURRENT_BINARY_DIR}/${test}.log \
-      && echo [PASS] ${test} \
-      || ( exit=$? ; \
-           echo [FAIL] ${test} ; \
-           cat ${CMAKE_CURRENT_BINARY_DIR}/${test}.log ; \
-           exit $exit )"
-    WORKING_DIRECTORY ${CMAKE_BINARY_DIR}
-    DEPENDS ${test}
-    VERBATIM)
-  add_custom_target(run-${test} DEPENDS ${test}.log)
+  if (WIN32)
+    # TODO: Figure out how to resolve this.
+    # Define dummy run-${test} target so add_dependencies still works
+    add_custom_target(run-${test})
```

Comment:
This is temporary. I need to better understand what the bash script is doing.

I wonder if a CMake script or python script can be used instead.